source/WHIMChannel.mc

```diff
@@ -89,11 +105,13 @@ class WHIMChannel extends Ant.GenericChannel
                             System.println( "EVENT_TX" );
                             break;
                         case Ant.MSG_CODE_EVENT_CHANNEL_CLOSED:
-                            System.println( "CHANNEL_CLOSED" );
-                            checkChannelClosure();
+                            System.println( "TIMER_EXPIRED_REOPEN_CHANNEL" );
+                            open();
```

Comment:
This is okay for now, but we won't always want to reopen when the channel closes. I have an idea for how we'll want to do this, so it may be best for me to make a separate commit for that (this commit is already pretty big anyway!).
We can talk more about this tomorrow though, along with everything else.

Reply:
Acknowledged.

We will add the solution we discussed in a future commit.